documentation/docs/deploy/installation.md

```diff
@@ -13,6 +13,8 @@ Integration Server based on Gradle and docker images. Therefore, you have to hav
 * Docker Compose  
 * Gradle 6+
 
+If you want to provision a kubernetes cluster on cloud then you have to have Terraform pre-installed on your machine.
```

Comment:
If you want to provision a Kubernetes cluster on cloud with help of terraform first you have to ensure that you have installed it. 

Something like that, it will be not the single option to do it.